mqtt/mqtt-edgehub/src/auth/authorization/policy.rs

```diff
@@ -0,0 +1,217 @@
+use thiserror::Error;
+use tracing::debug;
+
+use mqtt_broker::auth::{Activity, Authorization, Authorizer, Operation};
+use mqtt_policy::{MqttSubstituter, MqttTopicFilterMatcher, MqttValidator};
+use policy::{Decision, Policy, PolicyBuilder, Request};
+
+/// `PolicyAuthorizer` uses policy engine to evaluate the activity.
+///
+/// Policy definition comes from the Edge Hub twin. Before use, `PolicyAuthorizer` must be
+/// initialized with policy definition by calling `update` method.
```

Comment:
Where is the update method called? 

If we have the original policy at startup, why wouldn't we add it in to begin with?

Reply:
We don't have the policy at startup and right now no one calls `update` method. It would come from the edge hub twin, and that's the task I'm working on next.

I don't want to check in the code that reads the policy from the file, since that code is only for the demo, and not for prod :)